airbyte-ci/connectors/pipelines/pipelines/commands/airbyte_ci.py

```diff
@@ -180,6 +203,7 @@ def airbyte_ci(
 airbyte_ci.add_command(connectors)
 airbyte_ci.add_command(metadata)
 airbyte_ci.add_command(test)
+set_working_directory_to_root()
```

Comment:
Probably fine, but do any child-commands or forks need to be in a certain directory?

Reply:
Nope. Currently every `airbyte-ci` command is imported here and every airbyte-ci logic assumes `airbyte/` as the working directory.